backend/rules/rules.js

```diff
@@ -0,0 +1,119 @@
+const troopsRule = {
+  buildCost: 10,
+  buildSecond: 60,
+  level1: {
+    attack: 1,
+    defence: 1,
+    sustenance: 1,
+    upgradeCost: 20,
+    upgradeSecond: 120,
+  },
+  level2: {
+    attack: 2,
+    defence: 2,
+    sustenance: 1,
+    upgradeCost: 40,
+    upgradeSecond: 360,
+  },
+  level3: {
+    attack: 3,
+    defence: 3,
+    sustenance: 1,
+    upgradeCost: undefined,
+    upgradeSecond: undefined,
+  },
+};
+
+const buildingsRule = {
```

Comment:
```suggestion
const buildingsRule = {
  buildCost: 100,
  buildSecond: 60,
```

you can declare this in a constant at the beginning of the file, or here, but please don't write this to every level, this is a lot of code duplication